test/Macros/Inputs/wasm_plugin.swift

```diff
@@ -0,0 +1,210 @@
+import Foundation
```

Comment:
can we use Foundation in tests at this level? Specifically, this would need to be a copy of Foundation built for Wasm, so layering concerns might apply here. We only need it for JSON parsing so maybe it's a better idea to use the JSON coders in SwiftSyntax — which are already guaranteed to support Wasm targets by virtue of this whole endeavor. I've confirmed that SwiftSyntax.JSON does indeed work in this environment by copying the `JSON/*` source files over (the only source change I made was `s/_SwiftSyntaxCShims/WASILibc/g`) but I assume we'll want to cross-compile SwiftSyntax with CMake as a more robust approach. I haven't attempted to cross-compile it with CMake yet — is that something that should work out of the box?